projects/examples/src/components/datagrid/row-select-datagrid.example.component.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component } from '@angular/core';
+import { GridDataFetchResult, GridState, GridColumn, GridSelectionType } from '@vcd/ui-components';
+
+interface Data {
+    value: string;
+}
+
+/**
+ * A component that holds an example of the css classes per row capability.
+ */
+@Component({
+    selector: 'vcd-datagrid-row-select-example',
```

Comment:
Name of file should match your components, keeping datagrid in the beginning makes it easier to see things in a folder because of their sorting. This applies to any new files created